js/components/graph/Sidebar.js

```diff
@@ -87,10 +102,16 @@ export default class Sidebar extends React.Component {
             <div
               key={`active ${course}`}
               data-testid={`test ${course}`}
-              onClick={() => this.props.courseClick(course)}
               className="course-selection"
+              onClick={() => {
```

Comment:
This is good, but don't change the relative prop position (this prop used to be above `className`)